<p><strong>Author Response</strong></p>
<blockquote>
<p><strong>Reviewer #2 (Public Review):</strong></p>
</blockquote>
<blockquote>
<p>The authors present a viral dynamical model to predict the distribution of patient rebound times to bNAbs using only information about the population diversity at the onset of treatment. To parametrize this model, the authors identify mutational target sizes for bNAbs escape mutations from an analysis of deep mutational scanning data and infer the fitness costs of these mutations from a bNAb-free cohort. Paired with a rescaling factor that represents the amount of unsampled diversity in the reservoir, the authors have produced a model with few parameters that in aggregate does a good job of predicting trial outcomes in well-tracked cohorts. Using this validated model, they predict the percentage of late-rebounding viral populations treated with novel combination therapies, suggesting that three simultaneous bNAbs are required to prevent early rebound in the majority of individuals.</p>
</blockquote>
<blockquote>
<p>Strengths:</p>
</blockquote>
<blockquote>
<p>Because many of the model creation is largely driven by non-bNAb datasets, one of the major strengths is that the model is able to make predictions about rebound timing from very little data (i.e., population diversity before therapy). In doing so, it circumvents potential problems of overfitting limited data. In general, the analysis is careful and the authors derive many attributes from their data that important answer questions peripheral to the central stated goal. For example, they estimate the frequency of escape mutations arriving via mutation after therapy onset as opposed to those stemming from standing genetic variation before therapy onset. Additionally, they quantify the contribution of the unsampled genetic reservoir to escape dynamics.</p>
</blockquote>
<blockquote>
<p>The paper is clearly written, and will be an asset to newcomers to the field.</p>
</blockquote>
<p>We Thank the reviewer for the encouraging comments.</p>
<blockquote>
<p>Weaknesses:</p>
</blockquote>
<blockquote>
<ol>
<li>One potential weakness of the paper is that the model encodes all escape mutations as conferring a complete rescue effect in the presence of bNAbs. I didn’t see clear justification of this in the paper, and I’m not sure that evidence from the literature really suggests that this is true (or that is maybe only true for a subset of bNAbs). The IC50s of 3BNC117 to different viral isolates before and after treatment that are reported in the supplement of Caskey et al, 2015 show that there can be orders of magnitude differences in the evolved populations between individuals suggesting not all resistance is the same. The authors do not really consider that multiple smaller effect mutations combine to create larger effect escape phenotypes. While it’s possible that on these timescales, any viruses with positive growth rates should be sufficient to drive rapid population rebound and differences in these growth rates don’t matter, this argument wasn’t clearly articulated in the text.</li>
</ol>
</blockquote>
<p>This is an excellent point and we have added a new Appendix (Appendix 3) to the manuscript discussing this matter in detail. There are two related phenomena to consider: incomplete neutralization and the effect of multiple mutations to create an escape variant.</p>
<p>i. Regarding incomplete neutralization:</p>
<p>We have added language in the Discussion section, Limitations (page 12, Line 464-471):</p>
<p>"In our model of viral escape, we neglect the possibility of incomplete escape of the virus due to the reduced neutralization efficacy of bNAbs as their concentrations decay during trials. In Appendix 3, we show that this simplifying assumption is valid as long as the IC50 is not the same order of magnitude as the initial dosage concentration of the infused bNAb. Notably, the data from therapy trials used in this study fall into the regime for which we can neglect the impact of incomplete neutralization (Appendix 3-Figure 2). However, taking into account the dependence of viral fitness on bNAb concentration and its neutralization efficacy, as in the model proposed by [R34], could improve the long-term predictive power of our approach."</p>
<p>Moreover, in Appendix 3 we explore the effects of incomplete neutralization on rebound trajectories. As we show in Appendix 3-Figure 1, if an antibody has an IC50 against the viral variant which is an order of magnitude above the initial antibody concentration, the viral dynamics very closely follows the idealized “escaped” trajectory (i.e., with complete neutralization). On the other hand, for an IC50 an order of magnitude below the initial concentration, the viral dynamics behave similarly to a completely neutralized virus, with a late rebound (later than 8 weeks). We found that the most important effect of incomplete neutralization on the dynamics of viremia occurs when the antibody has an IC50 against a resistant variant that is roughly of similar magnitude to the initial bNAb concentration in a patient’s serum; see Appendix 3-Figure 1. In Appendix 3-Figure 2 we show the distribution of IC50 and the initial bNAb concentration from the 10-1074 trial [R35] to see how often we would expect IC50 and initial concentration to be of the same order of magnitude. We find that the IC50 values in this trial are much lower (higher) for susceptible (resistant) variants compared to the initial bNAb concentration in all patients. Therefore, our simplified model assuming that a viral variant is either fully resistant or susceptible to a bNAb (i.e., no incomplete escape) is a reasonable approach for capturing the statistics of treatment failure at the concentrations tested in these trials. Nonetheless, developing a genotype-to-neutralization model such as the ones in ref. [R1,R2] may allow for a more nuanced approach to characterize neutralization in future work.</p>
<p>ii. Regarding the effect of multiple mutations for escape:</p>
<p>One might still argue that the single-site substitution model of escape is limited, and that we have ignored the possibility of escape requiring multiple mutations from the consensus strain to generate a fully resistant and viable variant. However, viruses which require more than one mutation at treatment initiation cannot contribute to rebound because the neutralization timescale is too short for the virus to acquire more than one mutation. As we show in Figure 2-E, even acquiring one resistant mutation after infusion is rare. The de novo resistant population has frequency x(µ) ∼ 10−5 (equation 3, page 10). Requiring two independent mutation events instead of one would replace x(µ) with (x(µ))2 (roughly 10−10). Such a double-mutant population could never surpass the stochastic threshold for establishment, since the characteristic extinction frequency xext is on the order of 10−4 (Figure 2-D).</p>
<p>Although double-mutants are very unlikely be produced de novo in the course of a therapy, they could still be considered as present in the background genome. We may therefore view the question of multiple mutations as being closely related to the background dependence of escape pathways. As we have noted with regards to the shortcomings of the DMS data (see answer to Q#3 from essential revisions), our incomplete understanding of epistatic effects remains a limitation of our analysis and more data would be needed to address this problem.</p>
<blockquote>
<ol>
<li>The manuscript identifies a number of escape versus susceptible mutations based on DMS data and other patient-derived data taken from the literature. I remain incompletely convinced that these resistance mutations alone can explain population rebound in the clinical trial data that the authors fit. For example, for the trial on 3BNC117, this paper identifies four sites (279, 281, 282 and 459, listed in Appendix 1) where specific amino acid identities should confer resistance to 3BNC117. In looking at the genotypes of 10 viral populations treated with 3BNC117 and plotted in Figure 4 of this original paper (Caskey et al, 2015), only 1 of the 10 post-treatment viral populations has mutations at any of these four sites identified in this manuscript (279, 281, 282, 459). This suggests that the description of resistance mutations may not be sufficiently inclusive. The mutational target size is a critically important part of the model, so the potential for resistance outside of the identified ones could be problematic. Relating to the point above, these mutations may not have appeared in the screen for resistance mutations because they are of smaller effect. I would like the authors to try to demonstrate a better validation of their mutational targets.</li>
</ol>
</blockquote>
<p>There are two papers describing results from this trial:</p>
<p>(a) Caskey et al., Nature 2015 [R36] (the paper that the reviewer mentioned)</p>
<p>(b) Schoofs et al., Science 2016 [R37]</p>
<p>Sequences from some participants are included in only one the studies. We considered sequences derived from whole genome sequencing (available at GenBank), originated from Schoofs et al. [R37]. Sequences from Caskey et al. [R36] were generated usinghigh throughput bulk sequencing. Therefore, there is some discrepancy between the escape sites that we identify from the whole genome sequencing data of ref. [R37], and those identified in ref. [R36]. Nonetheless, we do find escape pathways in ref. [R36] which match the results we found from data originating in ref. [R37]. These include escape substitutions at HXB2 sites 279 (seen in patient 2A1), 281 (in patients 2E2, 2A3), and 459 (in patient 2C5). See Figure 4-A in ref. [R36] for more details.</p>
<p>We have fixed some references and added clarification in the Methods section: Data from bNAb trials (page 13 lines 516-518). The reviewer’s point is valid regarding the difficulties with the CD4bs bNAbs, and also that epistatic effects could influence the escape of a given variant on different genetic backgrounds. In the new appendix (Appendix 4) we discuss these issues in more detail; see our response to Q #3 of essential revisions.</p>
<blockquote>
<ol>
<li>Maybe relatedly, the authors identify that there are potential difficulties in using the DMS data from the CD4 binding site antibodies 3BNC and VRC01, and so they supplement this analysis of escape-mediating variants with other data sources (paragraph starting on line 490). First, it would be useful to have more detail around how exactly these mutations were identified from these other sources. Second, it sounds like the mutations identified in DMS for 3BNC and VRC01 aren’t concordant with those that are observed in treated HIV populations. I’m not familiar enough with these trials to know whether there is sufficiently extensive patient genetic data for each of these bNAbs treatments that can be used to look for large effect escape mutations, but it would be useful to have some measurement of how predictive these DMS-identified mutations are of actual patient escape mutations. Could comparing these two distributions (of DMS-identified mutations and patientidentified mutations) in cases in which both are available give us more confidence about their performance when only DMS data is available?</li>
</ol>
</blockquote>
<p>We agree with the reviewer that the DMS data required further consideration and we have added new Appendix (Appendix 4). Following the reviewer’s suggestion to compare the predictive results of the DMS data vs. trial, Appendix 4-Figure 1 now shows these predictions side-by-side. We also analyzed another trial with PGT121 for which we can now compare our predictions based on the escape variants inferred from the DMS data and from the trial data [R9]. Although it is difficult to generalize based on two bNAbs (10-1074 and PGT121), it seems that the DMS data may be slightly more optimistic, which accords with our intuition that the lack of diversity in the DMS parent strain may preclude background-dependent mutations which are more likely in-vivo. We stress that, in our view, the PGT121 predictions are remarkably good considering there are no new parameters fit. These issues are discussed in detail in Appendix 4, Discussion section, and the Methods sections; see a more detailed response to Q#3 of essential revisions.</p>
<blockquote>
<ol>
<li>It was not completely clear how the application of multiple bNAbs worked in the context of the model - did genotypes need to have one or more escape mutations for each bNAbs in order to replicate? For a three-bNAb combination therapy, is a virus carrying two escape mutations able to replicate?</li>
</ol>
</blockquote>
<p>We have attempted to clarify with a direct statement:</p>
<p>For multivalent treatment, a virus must be resistant to all antibodies comprising the treatment to have a positive growth after infusion. (page 4, line 162)</p>
<blockquote>
<ol>
<li>The paper was quite brief in terms of placing its own work in the context of other modeling studies of bNAbs escape.</li>
</ol>
</blockquote>
<p>We have substantially extended the introduction to discuss other studies on bNAb escape and to highlight our contributions in this work. Please see the answer to Q#1 of the essential revisions and also the red marked text in the introduction.</p>
<blockquote>
<ol>
<li>The manuscript analyzes the use of bNAbs for suppression of viral load, but does not discuss what the model might tell us about maintaining viral suppression in individuals with suppressed viral loads transitioning off of ART (which seems like it might be a more likely use case for bnABs in the future).</li>
</ol>
</blockquote>
<p>Thanks for pointing this out to us. We have added the following paragraph to the Discussion section (lines 442-455):</p>
<p>“One strategy to achieve a longer term treatment success is by combining bNAb therapy with ART. One main advantage of bNAb therapy is the fact that it can be administered once every few months, in contrast to ART, which should be taken daily and missing a dose could lead to viral rebound. Although multivalent bNAb therapy reduces the chances of short-term viral escape, viral escape remains a real obstacle for longer term success of a treatment with bNAbs. Alternatively, (fewer) bNAbs can be administered in combination with ART [R22,R23], whereby ART could lower the replication rate of the HIV population, reducing the viral diversity and the chances of viral escape. Specifically, we can expect that emergence and establishment of rare (i.e., strongly deleterious) escape variants against bNAbs to be less likely in ART+ patients, which suggests that fitness-limited bNAbs should be more effective in conjunction with ART. Still, more data would be necessary to understand the longterm efficacy of such augmented therapy, and specifically the role of viral reservoirs in this context. A modeling approach could then shed light on how ART administration and bNAb therapy could be combined to efficiently achieve viral suppression.”</p>
<blockquote>
<ol>
<li>This model assumes that the pressure imposed by bNAbs is constant for the first 8 weeks. What are the half-lives of the bNAbs involved, and is this a fair assumption? For example, Kwon et al, J Virol 2016 suggests 10E8 has a half-life of 5 days. Wouldn’t this require ongoing infusions to keep clinically relevant levels of the bNAbs around?</li>
</ol>
</blockquote>
<p>We thank the reviewer for pointing this out to us. Since submission we have learned that 10E8 is subject to several problems. It has displayed toxicity in trials, and this short half life might be consistent with selfbinding see trial report <a href="https://clinicaltrials.gov/ct2/show/NCT03565315">https://clinicaltrials.gov/ct2/show/NCT03565315</a>. In contrast the other antibodies we considered, including 10-1074 (half-life: 24.0 days in uninfected and 12.8 days in HIV-1-infected [R35]), 3BNC117 (half-life: about 18 days based on [R37]), PGT121 (half-life: about 20 days based on figure plots in [R9])), are likely to remain above the neutralizing threshold for the duration of the study. Indeed avoiding escape is not the end of the story, and successful bNAb therapy would target longer-lived antibodies. We now point out this caveat in the Discussion section: (lines 491-498)</p>
<p>“It should be noted that our analysis in [figure 4c (antibody ranking)] only focuses on one aspect of therapy optimization, i.e., the suppression of escape. Other factors, including potency (neutralization efficacy) and half-life of the bNAb, or the patient’s tolerance of bNAbs at different dosage should also be taken into account for therapy design. For example, the bNAb 10E8, which we identified as of the most promising mono-therapy candidates in Figure 4, is shown to be poorly tolerated by patients with short half-life [R24], making it undesirable for therapy purposes. Thus, the bNAb candidates shown in Figure 4C should be taken as a guideline to be complemented with further assessment of efficacy and safety for therapy design.”</p>
<blockquote>
<p><strong>Reviewer #3 (Public Review):</strong></p>
</blockquote>
<blockquote>
<p>The authors attempted to identify an optimal combination of broadly neutralizing antibodies (bNAbs) that can suppress escape of HIV-1 from the therapy. To do so, the authors fit a birth-death model of viral dynamics using published longitudinal HIV sequence data from 9 untreated patients. Using inferred quantities to parametrize the model subject to bNAb infusion, they predict the distribution of rebound times of HIV in therapy trials with two mono-therapy and their combination. Finally, using deep mutational scanning (DMS) data to identify escapemediating variants against 9 bnAbs for HIV, they propose a triplet combination that may best suppress early viral rebound. While the goal is clear, there are a number of major weaknesses that curtail the quality of the work:</p>
</blockquote>
<blockquote>
<ol>
<li>First, the approach is not novel. It at best represents a synthesis of known methods and published data sets.</li>
</ol>
</blockquote>
<p>We have now substantially extended and restructured the introduction section to highlight the novelty of our approach and compare our work to prior studies. In brief:</p>
<p>We discuss the modeling and machine learning techniques trained on experimental data from neutralization assays against pseudo-viruses to characterize the efficacy of bNAbs and their combinations against different variants of HIV [R1–R3]. These modeling approaches to optimization view the infection as a static collection of viral strains to be neutralized as opposed to an actively evolving population. We then discuss the mechanistic models that have been developed to explain the dynamics of viremia in patients following passive infusion of bNAbs [R4–R9]. These detailed models use trial data to fit parameters in relation to a bNAb’s efficacy in clearing virions, reducing viral load, etc. Although many of the inferred parameters are common across studies, these detailed mechanistic models cannot easily generalize from one trial to another in order to predict the efficacy of a new bNAb mono- or combination therapy. We discuss that evolution of the HIV population is another key factor to consider in modeling the dynamics of viremia in response to therapy with ART or bNAb. We then present our approach as a coarse-grained evolutionary model of viral response to bNAb infusion that uses genetic data of HIV in untreated patient to predict bNAb therapy outcome by characterizing the chances of viral escape from a given bNAb in patients. Although our model does not accurately reproduce the detailed dynamics of viremia in each patient and lacks the mechanistic insight of richer models proposed previously (these are not our goals), it can accurately predict the distribution of viral rebound times in response to passive bNAb infusions – a key measure of efficacy for a bNAb therapy trial. We then emphasize that our prediction for the viral rebound time in response to a bNAb relies on only a few patient-specific parameters (i.e., the genetic diversity of patients prior to treatment), and is primarily done based on the inferred genetic parameters from the deep sequencing of HIV-1 populations in a separate cohort of ART-naive patient. Therefore, we argue that our model could be used to guide therapy trial design by identifying optimal combinations of bNAbs to suppress evolutionary escape of HIV in patients.</p>
<p>The detailed added text is marked in red in the introduction.</p>
<p>In addition, in the Discussion section we have added the following text to argue how our approach can be used to identify combo therapies (lines 419-426):</p>
<p>“Combination therapy with more than two bNAbs (or drugs in ART) has long been shown to be more effective in suppressing early HIV rebound, both in theory and practice [R1,R2,R4,R10–R12]. In addition to corroborating this conclusion quantitatively, we provide a method for assessing new bnAbs for which escape mutations are known. Our method can be understood as a tool to navigate the combinatorial explosion of higher order cocktails for which we cannot possibly test all combinations. By assessing the evolvability of resistance against different combinations we can identify the best therapies to target for clinical trial. Specifically, we show that to suppress the chance of viral rebound to below 1%, we show that a combo-therapy with 3 bNAbs with a mixture of mutation- and selection-limited strategies that target different regions of the viral envelope is necessary. Such combination can counter the full variation of viral diversity observed in patients. We found that PG9, PG151, and VRC01, which respectively target V2 loop, Interface, and CD4 binding site of HIV envelope, form an optimal combination for a 3-bNAb therapy to limit HIV-1 escape in patients infected with clade B of the virus.”</p>
<p>Taken together, our main biological results can be summarized as follows:</p>
<p>(a) Predicting the distribution of bNAb treatment outcomes (short-term) for trials that have already been conducted (Figure 3-A)</p>
<p>(b) Estimates for all combinatorial possibilities for 9 bNAbs studied in this work. (Figures 3 and 4-D)</p>
<p>(c) Quantitative corroboration of importance of multivalent treatment and necessity of at least 3 bNAb’s in effective therapy (Figure 4-D)</p>
<p>(d) An identification and discussion of the importance of mutation-limited and selection-limited antibodies for therapy design (Figure 4-C)</p>
<p>(e) Quantifying the relative importance of de novo mutations vs. standing variation via simulations (Figure 2)
(f) Providing a quantitative approach for rational design of bNAb therapy</p>
<p>In addition to the new biological insight, this work presents methodological novelty, specifically with regards to robust algorithms for inference of evolutionary parameters and the statistical tests to quantify the accuracy of such inferences. These include (but not limited to):</p>
<p>(a) Inference of mutational target size from the nucleotide substitution pathways for acquiring resistance (Figure 1-D, and Equations 25 and 26)</p>
<p>(b) Bayesian posterior for the steady-state relative fitness values of resistant and susceptible variants from sequences of HIV populations in untreated patients (Equations 28-30 and Algorithm 4, Figure 4A and 4-supplement 2 for validation)</p>
<p>(c) Minimum-disparity based approach for measuring the robustness of the inferred selection parameters and a minimum-disparity based approach for hypothesis testing in the context of censored and categorical data, since rebound times may fall into the categories of right-censored late rebounds (&gt; 56 days) and no response (NR). (Equation 32 and Algorithm 5, and Figure 4-supplement 4)</p>
<p>(d) Quantifying the critical extinction threshold for therapy success (Equation 17 and Figure 3-D)</p>
<p>(e) Quantifying and inferring the contribution of the viral reservoir to treatment outcomes (Equation 35 Figure 3-E)</p>
<p>These statistical measures and modeling developments are likely to be helpful for inference of evolutionary parameters from sub-sampled genetic data in other evolving populations.</p>
<blockquote>
<ol>
<li>Second, the analyses and computational data cannot justify the major claims, in particular the prediction on optimal bNAb combinations - the central goal of this work. Specifically, match of rebound time distribution is only achieved for early rebound due to ineffective bNAbs. This limited validity under restrictive assumptions (within a limited time window) thus cannot validate the optimality of identified combinations that count on effective bNAbs for delayed rebound. More importantly, the proposed optimal combinations are highly sensitive to data quality and depth. In particular, DMS data cannot faithfully probe low-frequency variants that are chiefly responsible for rebound, which undermines the predictive power of the approach.</li>
</ol>
</blockquote>
<ol>
<li>With regards to the utility of bNAb therapy to suppress early viral rebound:</li>
</ol>
<p>We agree with the reviewer that the long-term efficacy of a bNAb treatment is important and we now discuss the nuances relevant to this issue in more depth in the manuscript; see the response to Q#4 of essential revisions. Nonetheless, we want to emphasize that our method can be understood as a tool to navigate the combinatorial explosion of higher order bNAb cocktails for which we cannot possibly test all combinations. By assessing the evolvability of resistance against different combinations we can identify the best candidate therapies that can be further tested in trials. In the manuscript we also discuss the potential role of augmenting bNAb therapies with ART to achieve a long-term success; see the response to Q#4 of essential revisions.</p>
<ol>
<li>With regards to the limited utility of DMS data to detect escape variants:</li>
</ol>
<p>We agree with the reviewer that the utility and limitations of the DMS data should be more clearly demonstrated. We have included a new Appendix (Appendix 4) to show the robustness of our predictions for rebound time distributions, when identifying escape sites from the DMS data versus the trial data. We performed this comparison for the 10-1074 and the PGT121 bNAbs, for which we have access to both the DMS and the trial data. Note that the analysis of the newly published therapy trial dataset with the PGT121 bNAb [R9] was added to the revised version of this manuscript. Please see the response to Q#3 of essential revisions for further details on our efforts to highlight the utility and limitations of DMS data for bNAb therapy design.</p>
<blockquote>
<ol>
<li>Third, the main results are already known from earlier work. It has been long known that a combination of more than two bnAbs is more effective in suppressing early rebound than fewer. Moreover, it has been shown recently that bnAb (VRC01) infusion acts to amplify pre-existing bnAb-resistant viral strains, leading to fast HIV rebound. Hence, it is unclear what new insight this work confers.</li>
</ol>
</blockquote>
<ol>
<li>With regards to combination therapy:</li>
</ol>
<p>As noted in response to Q# 1 of the reviewer, we agree that combination therapy with more than two bNAbs has been shown to be more effective in suppressing early HIV rebound. This work corroborates this conclusion quantitatively and provides a methods to assess the efficacy of new bNAb combinations o suppress viral rebound. For further details, please see our response to Q#1 of essential revisions.</p>
<ol>
<li>With regards to the role of pre-existing bNAb resistant variants in viral escape:</li>
</ol>
<p>One contribution of our work is in quantifying the role of pre-existing resistant variants for escape against bNAbs. Specifically, we used our evolutionary model to quantify the fraction of escape events driven by preexisting bNAb-resistant variants versus viral escape due to spontaneous mutation, and showed that mutationmediated escape accounted for less than 5% of escape; whether this fraction is 5% or 20% can be consequential for therapy design, yet difficult to infer from limited trial data. Although evidence from trial data (VRC01 study pointed by the reviewer) may be insightful, we believe that our quantitative assessment with mathematical modeling and evolutionary reasoning can be extended to a variety of bNAb studies to guide therapy design.</p>
<blockquote>
<ol>
<li>Lastly, suppression of early rebound alone is not a sufficient measure of therapy efficacy. Late rebound is not necessarily a sign of viral control, but might instead indicate selection for cross-resistant viral mutants - an even more detrimental outcome. In addition, this work has neglected bnAb dynamics or influence of infused bnAbs on the response of endogenous B cells, which will be essential for understanding viral dynamics, especially when infused bnAbs are relatively effective at suppressing early rebound.</li>
</ol>
</blockquote>
<ol>
<li><p>Regarding selecting for cross-resistant viral mutants:
All combination therapy are likely to select for cross-resistant viral mutants to some extent. To reduce the chances of emegence of cross-resistant variants, the therapy should keep viremia low to reduce the diversity of circulating strains. Although bNAbs (even in combinations) may be inefficient to suppress long term viral diversity, augmenting bNAb therapy with ART may be the solution to this problem. We are now emphasizing this issue in the Discussion section; see response to Q#4 of essential revisions.</p></li>
<li><p>Regarding the neglected bNAb dynamics:</p></li>
</ol>
<p>We have added a new Appendix 3 and also added language in the Discussion.</p>
<p>From the Discussion (Lines 464-472):</p>
<p>In our model of viral escape, we neglect the possibility of incomplete escape of the virus due to the reduced neutralization efficacy of bNAbs as their concentrations decay during trials. In Appendix 3, we show that this simplifying assumption is valid as long as the IC50 is not the same order of magnitude as the initial dosage concentration of the infused bNAb. Notably, the data from therapy trials used in this study fall into the regime for which we can neglect the impact of incomplete neutralization (Appendix 3-Figure 2). However, taking into account the dependence of viral fitness on bNAb concentration and its neutralization efficacy, as in the model proposed by [R34], could improve the long-term predictive power of our approach.</p>
<p>Moreover, in Appendix 3 we explore the effects of incomplete neutralization on rebound trajectories. As we show in Appendix 3-Figure 1, if an antibody has an IC50 against the viral variant which is an order of magnitude above the initial antibody concentration, the viral dynamics very closely follows the idealized “escaped” trajectory (i.e., with complete neutralization). On the other hand, for an IC50 an order of magnitude below the initial concentration, the viral dynamics behave similarly to a completely neutralized virus, with a late rebound (later than 8 weeks). We found that the most important effect of incomplete neutralization on the dynamics of viremia occurs when the antibody has an IC50 against a resistant variant that is roughly of similar magnitude to the initial bNAb concentration in a patient’s serum; see Appendix 3-Figure 1. In Appendix 3-Figure 2 we show the distribution of IC50 and the initial bNAb concentration from the 10-1074 trial [R35] to see how often we would expect IC50 and initial concentration to be of the same order of magnitude. We find that the IC50 values in this trial are much lower (higher) for susceptible (resistant) variants compared to the initial bNAb concentration in all patients. Therefore, our simplified model assuming that a viral variant is either fully resistant or susceptible to a bNAb (i.e., no incomplete escape) is a reasonable approach for capturing the statistics of treatment failure at the concentrations tested in these trials. Nonetheless, developing a genotype-to-neutralization model such as the ones in ref. [R1,R2] may allow for a more nuanced approach to characterize neutralization in future work.</p>
<ol>
<li>Regarding the endogenous B cell response:</li>
</ol>
<p>The reviewer’s point about the influence of the infused bNAbs on the response of endogenous B cells is very interesting. Unfortunately, we do not have data about these interactions and can only speculate about their relevance in bNAb therapy. Nonetheless, we should emphasize that our model does capture the impact of immune pressure on (pre-trial) viral populations in a coarse-grained way. The fitness values that we infer are not determined in vacuum: they are derived from allele frequencies measured in evolving HIV-1 populations under the constantly changing immune challenge from a host’s endogenous immune system. Therefore, it is reasonable to assume that the changes in the immune challenge are to some extent accounted for in our inference of fitness costs for escape variants. The enormous variation in human immune response means that we can only expect to model these effects in aggregate — as an average over the range of immune systems we find in a dataset. But that we have a mechanism to capture these effects at all is likely a major contributor to the success of our predictions in clinical trials.</p>