<p><strong>Reviewer #2 (Public Review):</strong></p>
<p>The authors present a viral dynamical model to predict the distribution of patient rebound times to bNAbs using only information about the population diversity at the onset of treatment. To parametrize this model, the authors identify mutational target sizes for bNAbs escape mutations from an analysis of deep mutational scanning data and infer the fitness costs of these mutations from a bNAb-free cohort. Paired with a rescaling factor that represents the amount of unsampled diversity in the reservoir, the authors have produced a model with few parameters that in aggregate does a good job of predicting trial outcomes in well-tracked cohorts. Using this validated model, they predict the percentage of late-rebounding viral populations treated with novel combination therapies, suggesting that three simultaneous bNAbs are required to prevent early rebound in the majority of individuals.</p>
<p>Strengths:</p>
<p>Because many of the model creation is largely driven by non-bNAb datasets, one of the major strengths is that the model is able to make predictions about rebound timing from very little data (i.e., population diversity before therapy). In doing so, it circumvents potential problems of overfitting limited data. In general, the analysis is careful and the authors derive many attributes from their data that important answer questions peripheral to the central stated goal. For example, they estimate the frequency of escape mutations arriving via mutation after therapy onset as opposed to those stemming from standing genetic variation before therapy onset. Additionally, they quantify the contribution of the unsampled genetic reservoir to escape dynamics.</p>
<p>The paper is clearly written, and will be an asset to newcomers to the field.</p>
<p>Weaknesses:</p>
<p>One potential weakness of the paper is that the model encodes all escape mutations as conferring a complete rescue effect in the presence of bNAbs. I didn't see clear justification of this in the paper, and I'm not sure that evidence from the literature really suggests that this is true (or that is maybe only true for a subset of bNAbs). The IC50s of 3BNC117 to different viral isolates before and after treatment that are reported in the supplement of Caskey et al, 2015 show that there can be orders of magnitude differences in the evolved populations between individuals suggesting not all resistance is the same. The authors do not really consider that multiple smaller effect mutations combine to create larger effect escape phenotypes. While it's possible that on these timescales, any viruses with positive growth rates should be sufficient to drive rapid population rebound and differences in these growth rates don't matter, this argument wasn't clearly articulated in the text.</p>
<p>The manuscript identifies a number of escape versus susceptible mutations based on DMS data and other patient-derived data taken from the literature. I remain incompletely convinced that these resistance mutations alone can explain population rebound in the clinical trial data that the authors fit. For example, for the trial on 3BNC117, this paper identifies four sites (279, 281, 282 and 459, listed in Appendix 1) where specific amino acid identities should confer resistance to 3BNC117. In looking at the genotypes of 10 viral populations treated with 3BNC117 and plotted in Figure 4 of this original paper (Caskey et al, 2015), only 1 of the 10 post-treatment viral populations has mutations at any of these four sites identified in this manuscript (279, 281, 282, 459). This suggests that the description of resistance mutations may not be sufficiently inclusive. The mutational target size is a critically important part of the model, so the potential for resistance outside of the identified ones could be problematic. Relating to the point above, these mutations may not have appeared in the screen for resistance mutations because they are of smaller effect. I would like the authors to try to demonstrate a better validation of their mutational targets.</p>
<p>Maybe relatedly, the authors identify that there are potential difficulties in using the DMS data from the CD4 binding site antibodies 3BNC and VRC01, and so they supplement this analysis of escape-mediating variants with other data sources (paragraph starting on line 490). First, it would be useful to have more detail around how exactly these mutations were identified from these other sources. Second, it sounds like the mutations identified in DMS for 3BNC and VRC01 aren't concordant with those that are observed in treated HIV populations. I'm not familiar enough with these trials to know whether there is sufficiently extensive patient genetic data for each of these bNAbs treatments that can be used to look for large effect escape mutations, but it would be useful to have some measurement of how predictive these DMS-identified mutations are of actual patient escape mutations. Could comparing these two distributions (of DMS-identified mutations and patient-identified mutations) in cases in which both are available give us more confidence about their performance when only DMS data is available?</p>
<p>It was not completely clear how the application of multiple bNAbs worked in the context of the model - did genotypes need to have one or more escape mutations for each bNAbs in order to replicate? For a three-bNAb combination therapy, is a virus carrying two escape mutations able to replicate?</p>
<p>The paper was quite brief in terms of placing its own work in the context of other modeling studies of bNAbs escape.</p>
<p>The manuscript analyzes the use of bNAbs for suppression of viral load, but does not discuss what the model might tell us about maintaining viral suppression in individuals with suppressed viral loads transitioning off of ART (which seems like it might be a more likely use case for bnABs in the future).</p>
<p>This model assumes that the pressure imposed by bNAbs is constant for the first 8 weeks. What are the half-lives of the bNAbs involved, and is this a fair assumption? For example, Kwon et al, J Virol 2016 suggests 10E8 has a half-life of 5 days. Wouldn't this require ongoing infusions to keep clinically relevant levels of the bNAbs around?</p>